realm/realm-library/src/androidTest/java/io/realm/internal/CollectionTests.java

```diff
@@ -258,7 +260,7 @@ public void addListener_shouldBeCalledToReturnTheQueryResults() {
         looperThread.keepStrongReference(collection);
         collection.addListener(collection, new RealmChangeListener<Collection>() {
             @Override
-            public void onChange(Collection collection1) {
+            public void onChange(@Nonnull Collection collection1) {
```

Comment:
Isn't it enough to mark the RealmChangeListener interface with @Nonnull ? Doing it at every call site seems a bit excessive?